### ISSUE TITLE
Authenticating against multiple SalesForce organisations - can it be done?

### ISSUE THREAD

**Issue by blorange2** [roblesterjr04/EloquentSalesForce#110]
Hello,

I'm in a situation where I'd like to send data from one Salesforce organisation to another after a certain point so I'd like to have a specific model like `OtherOrganisationLead` and authenticate against another organisation when working with it.

I see in https://github.com/omniphx/forrest/tree/master?tab=readme-ov-file#soap-authentication-flow that if you use SOAP you can login to multiple accounts.

Is there a way to authenticate against different accounts on the fly?

**Comment by roblesterjr04** [comment]
Yeah, you should be able to do it by changing the config on the fly per laravel methods.

**Comment by blorange2** [comment]
> Yeah, you should be able to do it by changing the config on the fly per laravel methods.

How does this work with models and the like? As a different Salesforce org may have different objects.

Esse

**Comment by blorange2** [comment]
How would this work with differing models? Essentially I have a point where I want to write an object's data to another object in a different Salesforce account.
